src/TypedData/StrawberryfieldFlavorDataDefinition.php

```diff
@@ -22,9 +22,9 @@ public function getPropertyDefinitions() {
       $info = &$this->propertyDefinitions;
       $info['item_id'] = DataDefinition::create('string')->setLabel('Item ID');
       $info['sequence_id'] = DataDefinition::create('string')->setLabel('Sequence ID');
-      $info['parent_id'] = DataReferenceTargetDefinition::create('integer');
+      $info['parent_id'] = DataReferenceTargetDefinition::create('integer')->setLabel('Parent Node ID');
       $info['target_id'] = DataReferenceDefinition::create('entity')
-      ->setLabel('Parent Node ID')
+      ->setLabel('Parent Node')
```

Comment:
Tiny change. Its the actual Node that people will see to changed to Parent Node and move the label Node ID to the other property